README.md

```diff
@@ -0,0 +1,29 @@
+# このアプリについて
+  - フィヨルドブートキャンプというプログラミングスクールで作成したSinatraのメモアプリです。
+  - 簡単なメモの新規投稿、一覧表示、編集、削除ができます。
+# アプリ立ち上げ方法
+## git clone
+作業したいローカルフォルダに移動します。
+`git clone https://github.com/cest-megu/fjorrd_sinatra_app.git`
+を実行します。
+
+## gemファイルのインストール
+- `fjord_sinatra_app`のディレクトリに移動します。
+- アプリ立ち上げのため、必要なgemファイルをインストールするために、
+```
+bundle install --path vendor/bundle
```

Comment:
`.bundle/config`の設定がされているので、この`--path vendor/bundle`のオプションは不要だと思います。詳しくは`bundle install --path vendor/bundle`実行時に表示される下記のメッセージを参照してください。

```ruby
[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local path 'vendor/bundle'`, and stop using this flag
```

それから、もしここまで理解した上でやられているのでしたら申し訳ないのですが、そもそも`vendor/bundle`は必要か？みたいな話もありますので良かったらこちらの記事などをみて見てください 👍 
https://qiita.com/jnchito/items/99b1dbea1767a5095d85

Reply:
了解しました！よくわからずに`--path vendor/bundle`を使っていたことがあったので、
記事を読んで取り急ぎ今は必要ないと判断したため、削除させていただきました！